packages/round-manager/src/features/round/__tests__/ViewRoundPage.test.tsx

```diff
@@ -237,26 +246,74 @@ describe("View Round", () => {
-          wrapWithReadProgramContext(
-            wrapWithRoundContext(<ViewRoundPage />, {
-              data: [mockRoundData],
-              fetchRoundStatus: ProgressStatus.IS_SUCCESS,
-            }),
-            { programs: [] }
-          ),
-          {
-            applications: [],
-            isLoading: false,
-          }
+  describe("display funding admin tab", () => {
+    it("displays No Information Available before round end date", async () => {
+      const roundEndTime = faker.date.future();
+      mockRoundData = makeRoundData({ roundEndTime });
+      render(
+        wrapWithBulkUpdateGrantApplicationContext(
+          wrapWithApplicationContext(
+            wrapWithReadProgramContext(
+              wrapWithRoundContext(<ViewRoundPage />, {
+                data: [mockRoundData],
+                fetchRoundStatus: ProgressStatus.IS_SUCCESS,
+              }),
+              { programs: [] }
+            ),
+            {
+              applications: [],
+              isLoading: false,
+            }
+          )
         )
-      )
-    );
-    const fundingAdminTab = screen.getByTestId("funding-admin");
-    fireEvent.click(fundingAdminTab);
-    expect(screen.getByText("No Information Available")).toBeInTheDocument();
+      );
+      const fundingAdminTab = screen.getByTestId("funding-admin");
+      fireEvent.click(fundingAdminTab);
+      expect(screen.getByText("No Information Available")).toBeInTheDocument();
+    });
+
+    it("displays matching stats table in funding admin page after round end date", async () => {
+      (useRoundMatchData as jest.Mock).mockImplementation(() => ({
+        return: {
+          data: [makeQFDistribution(), makeQFDistribution()],
+          error: null,
+          loading: false,
+        },
+      }));
+
+      const roundEndTime = faker.date.recent();
+      const roundStartTime = faker.date.past(1, roundEndTime);
+      const applicationsEndTime = faker.date.past(1, roundStartTime);
+      const applicationsStartTime = faker.date.past(1, applicationsEndTime);
+
+      const approvedProjects = [
+        makeApprovedProjectData(),
+        makeApprovedProjectData(),
+        makeApprovedProjectData(),
+      ];
+      mockRoundData = makeRoundData({
+        applicationsStartTime,
+        applicationsEndTime,
+        roundStartTime,
+        roundEndTime,
+        approvedProjects,
+      });
+      render(
+        wrapWithBulkUpdateGrantApplicationContext(
+          wrapWithApplicationContext(
+            wrapWithReadProgramContext(
+              wrapWithRoundContext(<ViewRoundPage />, {
+                data: [mockRoundData],
+                fetchRoundStatus: ProgressStatus.IS_SUCCESS,
+              })
+            )
+          )
+        )
+      );
+      const fundingAdminTab = screen.getByTestId("funding-admin");
+      fireEvent.click(fundingAdminTab);
+      expect(screen.getByText("Finalized Matching Stats")).toBeInTheDocument();
```

Comment:
@thelostone-mc as discussed, the test is failing at this line. Please take a look when you get time.